src/main/scala/FbEvents.scala

```diff
@@ -6,17 +6,22 @@ import scalaj.http._
 
 object FbEvents {
   case class FbEvent(id: String, name: String, description: Option[String], startTime: ZonedDateTime, endTime: Option[ZonedDateTime]);
-  case class Response(data: List[FbEvent]);
+  case class Paging(next: Option[String]);
+  case class Response(data: List[FbEvent], paging: Option[Paging]);
 }
 
 trait FbEvents extends FbJsonMarshalling {
   import FbEvents._
 
-  def getEvents(token: String, pageId: String): List[FbEvent] = {
-    val url = s"https://graph.facebook.com/v2.7/$pageId/events?access_token=$token";
+  def getEvents(url: String): List[FbEvent] = {
     Http(url).asString match {
       case HttpResponse(body, 200, _) =>
-        body.parseJson.convertTo[Response].data
+        val response : Response = body.parseJson.convertTo[Response];
```

Comment:
Oh and the explicit `: Response` isn't really needed here, seems clear enough without it